controllers/components/dashboard/dashboard_controller.go

```diff
@@ -91,7 +90,6 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 		WithAction(initialize).
 		WithAction(devFlags).
 		WithAction(configureDependencies).
-		WithAction(security.NewUpdatePodSecurityRoleBindingAction(serviceAccounts)).
```

Comment:
is `NewUpdatePodSecurityRoleBindingAction` is not more in use, we can probably remove it